README.md

```diff
@@ -402,6 +403,13 @@ ESLint rules:
 - [`space-infix-ops`](https://eslint.org/docs/rules/space-infix-ops.html)
 - [`comma-spacing`](http://eslint.org/docs/rules/comma-spacing)
 - [`no-multiple-empty-lines`](http://eslint.org/docs/rules/no-multiple-empty-lines)
+- [`max-len`](http://eslint.org/docs/rules/max-len)
+- [`max-lines-per-function`](http://eslint.org/docs/rules/max-lines-per-function)
+
+SonarQube rules:
+
+- `Lines should not be too long`
```

Comment:
This is confusing: is it line length, as in columns, or number of lines, as in rows. I would use something like the following:

>  Limit of the number of lines in a function to keep it small and focused.

Reply:
I agree it's not a great name but this is the actual name in SonarQube.  I've added a link to the documentation from the SonarQube website.  It is line length in terms of columns though :)